python/hail/docs/api.rst

```diff
@@ -28,6 +28,7 @@ Use ``import hail as hl`` to access this functionality.
     types <types>
     functions <functions/index>
     aggregators <aggregators>
+    scan<scan>
```

Comment:
note `scans <scans>` is more consistent (I'll do full review after you address Tim's comments)